testsuite/ext-tools/maintenance_json_generator.py

```diff
@@ -51,6 +51,8 @@
     "rocky9_minion": "/SUSE_Updates_EL_9-CLIENT-TOOLS_x86_64/",
     "alma9_minion": "/SUSE_Updates_EL_9-CLIENT-TOOLS_x86_64/",
     "oracle9_minion": "/SUSE_Updates_EL_9-CLIENT-TOOLS_x86_64/",
+    "slemicro51_minion": ["/SUSE_Updates_SLE-Manager-Tools-For-Micro_5_x86_64/",
```

Comment:
is this version correct and not missing a 1 after 5? just double checking

Reply:
Yes it's the "general" version repo. It's like the corresponding repo for "/SUSE_Updates_SLE-Manager-Tools_15_x86_64/", the general client tools repo for the OS instead of being SP specific